src/views/reverse/steps/lockingFunds.js

```diff
@@ -31,14 +26,12 @@ const LockingFunds = ({ classes, swapInfo, swapResponse }) => {
         to receive, this is important to keep the <br />
         swap atomic and trustless. It might take <br />
         up to 10 minutes. Please be patient. <br />
-        <a href={link}>Click here</a> to see the lockup transaction.
```

Comment:
Although I agree with #113 I do think that we should keep the block explorer link on this page. For the sake of verifying that the transaction was sent and not waiting for a transaction that doesn't exist to be confirmed.

Reply:
@michael1011 So we should add the link on both pages?

Reply:
Yes, I would say so.